### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by girma336**
###### In this task, we  working with the real live data from the SpaceX API. our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
- We added every thing with their give requirement

### CODE REVIEW

File: src/Redux/rocketSlice.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const rockSlice = createSlice({
+  name: 'rocket',
+  initialState: {
+    rockets: [],
+  },
+  reducers: {
+    bookRocket(state, action) {
+      const newId = action.payload;
+      // eslint-disable-next-line
```
Comment (Deepakdanger) [inline src/Redux/rocketSlice.js]: - [ ] Kindly remove this line & solve linter manually

File: src/Redux/rocketSlice.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      state.rockets = state.rockets.map((rocket) => {
+        if (rocket.id !== newId) return rocket;
+        return { ...rocket, reserved: true };
+      });
+    },
+    removeRocket(state, action) {
+      const newId = action.payload;
+      // eslint-disable-next-line
```
Comment (Deepakdanger) [inline src/Redux/rocketSlice.js]: - [ ] Kindly remove this line & solve linter manually

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @girma336  & @Ibtisam34 , 

Good job so far! 👏
GOOD UI DESIGN
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { NavLink } from 'react-router-dom';
+import logo from '../logo/planet.png';
+import './Navbar.css';
+
+function NavBar() {
+  return (
+    <nav className="navbar">
+      {/* <Link "> */}
```
Comment (rustam-tolipov) [inline src/components/Navbar.js:8]: [OPTIONAL] Please ensure that any unused commented codes are removed from the file. Thank you.

File: src/tests/Navbar.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import NavBar from '../components/Navbar';
+
+describe('Test snapshot componet of rocket', () => {
+  it('Renders correctly of nav bar', () => {
+    const tree = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <NavBar />
+        </Provider>
+      </BrowserRouter>,
```
Comment (rustam-tolipov) [inline src/tests/Navbar.test.js:15]: [OPTIONAL] You have implemented the snapshot tests well, however, we recommend also incorporating unit and integration tests into your project.

**Review by rustam-tolipov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
_Kindly check the comments belove_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**